tests/Behat/Resources/services.xml

```diff
@@ -1,6 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 
 <container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <imports>
+        <import resource="./legacy.php"/>
+    </imports>
+
```

Comment:
I'm wondering, wouldn't it be better if we put it just in `services` directory and call the file `mailer.php` or something like that 🤔.  I don't have a strong opinion, but I have mixed feelings about this.

Reply:
Agree, it'll be better, thanks 👍️